src/codeu/chat/server/Server.java

```diff
@@ -15,12 +15,11 @@
 
 package codeu.chat.server;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
+import java.io.*;
```

Comment:
I fixed this in my settings. Now it only does wild imports after a count of 50 instead of 5 😆 .